pkg/kubecost/summaryallocation.go

```diff
@@ -822,7 +822,7 @@ func (sas *SummaryAllocationSet) AggregateBy(aggregateBy []string, options *Allo
 
 			var key string
 			if options.IdleByNode {
-				key = ia.Properties.Node
+				key = fmt.Sprintf("%s/%s", ia.Properties.Cluster, ia.Properties.Node)
```

Comment:
I believe this was the cause of the reported error. `cluster/node` is used in all of the other key creation in this function if `idleByNode` is true. Here, it was just `node`.

Reply:
Argh, yes, good catch.